kornia/geometry/conversions.py

```diff
@@ -159,7 +159,7 @@ def convert_points_from_homogeneous(points: torch.Tensor, eps: float = 1e-8) ->
     # follow the convention of opencv:
     # https://github.com/opencv/opencv/pull/14411/files
     mask: torch.Tensor = torch.abs(z_vec) > eps
-    scale = torch.where(mask, 1.0 / (z_vec + eps), torch.ones_like(z_vec))
+    scale = torch.where(mask, torch.ones_like(z_vec) / (z_vec + eps), torch.ones_like(z_vec))
```

Comment:
```suggestion
    ones = torch.ones_like(z_vec)
    scale = torch.where(mask, ones / (z_vec + eps), ones)
```
besides, why broadcasting is not working here ?

Reply:
The broadcasting works fine but 1 / (z_vec + eps)  will promote to float32, probably better to keep it that way and convert back?

Reply:
Since this is a quite dire function that we use across many other functionality, I’d say that this should be benchmarked to see what what’s most expensive to cast whole result or allocate a big chunk of ones. @lferraz ?